README.md

```diff
@@ -0,0 +1,109 @@
+# OpenCompose
+
+[![Build Status](https://travis-ci.org/redhat-developer/opencompose.svg?branch=master)](https://travis-ci.org/redhat-developer/opencompose)
+
+The goal of OpenCompose is to make it easier to on-board developers to Kubernetes. It is a curated higher level abstraction for specific Kubernetes resources. Very simple idea, isn't it? Developer shouldn't have to learn various Kubernetes concepts just to test and deploy their applications on Kubernetes. The developer is generally concerned with the application that is being developed.
+
+We are at a very nascent stage of this project and we have listed some of our ideas as [issues](https://github.com/redhat-developer/opencompose/issues) and some of our ideas have taken shape in form of [examples](https://github.com/redhat-developer/opencompose/blob/master/examples/) but we are definitely looking for suggestions and contributions from the Kubernetes community. Please send PRs to improve this project, file issues and RFEs against this repository.
+
+### Installation
+We don't have a release yet, but you can easily build OpenCompose tool from source.
+All you need is [Go installed](https://golang.org/dl/) on your system. Than simply run:
+```
+go get -u https://github.com/redhat-developer/opencompose
+```
+
+### Usage
+```
+~$ opencompose --help
+opencompose is a tool to transform OpenCompose files into Kubernetes (and OpenShift) definitions
+
+Find more information at https://github.com/redhat-developer/opencompose.
+
+Usage:
+  opencompose [flags]
+  opencompose [command]
+
+Available Commands:
+  completion  Generate the opencompose completion code for bash
+  convert     Converts OpenCompose files into Kubernetes (and OpenShift) artifacts
+  validate    Print validate information
+  version     Print version information
+
+Use "opencompose [command] --help" for more information about a command.
+```
+
+### Example
+1) Create or download [hello-nginx.yml](https://github.com/redhat-developer/opencompose/blob/master/examples/hello-nginx.yaml) file.
+
+```yaml
+version: 0.1-dev
+services:
+- name: helloworld
+  containers:
+  - image: tomaskral/nonroot-nginx
+    ports:
+    - port: 8080
+```
+
+2) Convert OpenCompose file to Kubernetes objects
+```
```

Comment:
It seems too confusing to me to embed it here. I don't think that users would really care about it and the generated content is a subject of a change.